roles/wordpress-setup/tasks/database.yml

```diff
@@ -11,6 +11,7 @@
 - name: Create/assign database user to db and grant permissions
   mysql_user: name="{{ item.value.env.db_user }}"
               password="{{ item.value.env.db_password }}"
+              append_privs=true
```

Comment:
Yeah that's inconsistent on their part :( But they always use `yes`/`no` in the options section of their docs.